### PR TITLE
feat(curriculum): adding teacher bot workshop

### PR DESCRIPTION
**Pull request by jdwilkin4**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/workshop-teacher-chatbot/66b59829dba144ff1351220f.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+```js
+const name = "John";
+
+// "Hello, John!"
+`Hello, ${name}!`;
+```
+
+Start by creating a variable called `greeting`.
+
+Then using template literal syntax, assign a string that says `My name is` followed by the `botName` variable followed by a period (`.`).
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-teacher-chatbot/66b59829dba144ff1351220f.md:23]:
```suggestion
Next, using template literal syntax, assign a string that says `My name is`, followed by the `botName` variable, and ending with a period (`.`).
```

File: curriculum/challenges/english/25-front-end-development/workshop-teacher-chatbot/66b59b12a745e10011158f7b.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
+---
+id: 66b59b12a745e10011158f7b
+title: Step 4
+challengeType: 1
+dashedName: step-4
+---
+
+# --description--
+
+The next step is to create a few more variables that you will use in future bot messages.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-teacher-chatbot/66b59b12a745e10011158f7b.md:10]:
```suggestion
The next step is to create a few more variables that will be used in future bot messages.
```

File: curriculum/challenges/english/25-front-end-development/workshop-teacher-chatbot/66b59be6ab830800c4df9146.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+---
+id: 66b59be6ab830800c4df9146
+title: Step 5
+challengeType: 1
+dashedName: step-5
+---
+
+# --description--
+
+Now it is time to use the variables you created in the previous step.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-teacher-chatbot/66b59be6ab830800c4df9146.md:10]:
```suggestion
Now, it's time to use the variables you created in the previous step.
```

File: curriculum/challenges/english/25-front-end-development/workshop-teacher-chatbot/66b6e62423e8a031d6c1f03d.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+You can use template literals inside `console` statements like this:
+
+```js
+const developer = "Jessica";
+console.log(`Hello, my name is ${developer}.`);
+```
+
+Start by outputting the message `Here is an example of using the length property on the word [topic].` to the console. 
+
+Remember to replace `[topic]` with the `topic` variable and use proper template literal syntax like you did in the previous steps.
```
Comment (Sembauke) [inline curriculum/challenges/english/25-front-end-development/workshop-teacher-chatbot/66b6e62423e8a031d6c1f03d.md:21]:
```suggestion
Remember to replace `[topic]` with the `topic` variable, and use proper template literal syntax as you did in the previous steps.
```

**Review by Sembauke — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/25-front-end-development/workshop-teacher-chatbot/66b6fdb76441c738719039fa.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
+---
+id: 66b6fdb76441c738719039fa
+title: Step 13
+challengeType: 1
+dashedName: step-13
+---
+
+# --description--
+
+In the lecture videos you learned how to access the last character in a string like this:
```
Comment (Ksound22) [inline curriculum/challenges/english/25-front-end-development/workshop-teacher-chatbot/66b6fdb76441c738719039fa.md:10]:
```suggestion
In the lecture videos, you learned how to access the last character in a string like this:
```

File: curriculum/challenges/english/25-front-end-development/workshop-teacher-chatbot/66b70b8a611cbf3bcc5c6c5f.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
+---
+id: 66b70b8a611cbf3bcc5c6c5f
+title: Step 16
+challengeType: 1
+dashedName: step-16
+---
+
+# --description--
+
+In the previous lecture videos you learned how to work with the `indexOf` method like this:
```
Comment (Ksound22) [inline curriculum/challenges/english/25-front-end-development/workshop-teacher-chatbot/66b70b8a611cbf3bcc5c6c5f.md:10]:
```suggestion
In the previous lecture videos, you learned how to work with the `indexOf` method like this:
```

**Review by Ksound22 — APPROVED** [review]
Things are good, other than a few nitpicks with commas.

**Review by Sembauke — APPROVED** [review]

**Pull request merged**